src/app.js

```diff
@@ -1,68 +1,80 @@
--- Stap 1. Installeer en importeer Axios;
+//Stap 1. Installeer en importeer Axios;
 import axios from 'axios'
 
 console.log('Hallo daar!');
 
 // 1. Schijf een async. functie (EdHub FE JS 10.6)
 
 // Stap 3. Schrijf een asynchrone functie die, met behulp van Axios, een GET-request maakt naar het juiste endpoint. Log de response in de console en bestudeer de data goed.
-let countries = [];
+// let countries = [];
 async function getCountries(){
+    const countryList = document.getElementById("countries-list");
     try {
         const result = await axios.get('https://restcountries.com/v2/all');
-        // result info uti de API
+        const countries = result.data;
+        // result info uit de API
         // return result gaat dat niet goed
         // opbouwen data hier
         // return result.data;
-        console.log(result.data)
+        // Stap 4. Probeer eens om de _naam_ van het _allereerste_ land te loggen in de console, welk pad moet je hiervoor volgen?
+        console.log(result.data);
         // logArray(result.data);
+
+
+        //sorteren op Population
+        countries.sort((a, b) => {
+            return a.population - b.population;
+        });
+
+        result.data.map((country) => {
```

Comment:
Hier gebeurt iets raars. Je hebt bewust een array countries aangemaakt en deze gesorteerd. Je gaat hier echter verder met result.data. Het was logischer geweest met de gesorteerde 'countries' verder te gaan. Leest in ieder geval gemakkelijker.